packages/client/hmi-client/src/services/simulation.ts

```diff
@@ -43,3 +45,42 @@ export const parseSimulationPlan = (plan: SimulationPlan) => {
 	});
 	return { nodes, edges };
 };
+
+/**
+ * Given a simulation plan wiring diagram, convert to an IGraph representation
+ * for the renderer
+ */
+export const parseSimulationPlan2IGraph = (plan: SimulationPlan) => {
+	const graph = parseSimulationPlan(plan);
+	const g: IGraph<any, any> = { nodes: [], edges: [] };
+
+	// Initialize
+	graph.nodes.forEach((d) => {
+		g.nodes.push({
+			id: d.id,
+			label: d.id, // simulation plan currently allows duplicate labels which create tracking problems in the renderer
+			x: 0,
+			y: 0,
+			width: 40,
+			height: 40,
+			nodes: [],
+			data: {
+				boxType: d.boxType,
+				label: d.name
+			}
+		});
+	});
+
+	graph.edges.forEach((d, i) => {
+		console.log('edge >>', d);
```

Comment:
i assume this is meant to be here? feel free to close if it is.

Reply:
will remove